scripts/download

```diff
@@ -24,7 +24,7 @@ rm -rf ${CONTAINERD_DIR}
 mkdir -p ${CHARTS_DIR}
 mkdir -p ${DATA_DIR}
 
-curl --compressed -sfL https://github.com/k3s-io/k3s-root/releases/download/${VERSION_ROOT}/k3s-root-${ARCH}.tar | tar xf - --exclude=bin/socat
+curl --compressed -sfL https://github.com/rancher/k3s-root/releases/download/${VERSION_ROOT}/k3s-root-${ARCH}.tar | tar xf - --exclude=bin/socat
```

Comment:
I think the URL references are good to leave changed. It's just the Go package names that need to remain `rancher/k3s` for the time being.

Reply:
prefer to do all in one PR later than a 50/50 state. pls to just back em out and let's actually do this 100% when the discussion/design is approved

Reply:
You didn't actually change these, don't un-change them.

Reply:
ah i see your point. ill fix